We see a FTBFS for Python 3.13 if we leave this `wheel.py-api` setting in place.

diff --git a/pyproject.toml b/pyproject.toml
index a1db878..db3bf4c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -67,11 +67,6 @@ sdist.exclude = [
     "third-party/faiss/tutorial"
 ]
 
-[[tool.scikit-build.overrides]]
-if.platform-system = "^(darwin|linux)"  # NOTE: Windows ABI3 results in broken DLLs.
-if.abi-flags = "^$"  # NOTE: Free-threaded Python does not support stable ABI.
-wheel.py-api = "cp39"
-
 [tool.cibuildwheel]
 archs = "native"
 # NOTE: We do not support cp39 and cp310 on Windows ARM64
